### DISCUSSION TITLE
Allow ajax calls to be filtered

### DISCUSSION THREAD

**Discussion by brunobaudry** [SafeExamBrowser/seb-win-refactoring#930] [Q&A]
Seb does not block ajax called URL, that would be very helpfull.

**Comment by dbuechel** [comment]
You can block all requests, see Network > Filter > Filter also embedded content: https://safeexambrowser.org/windows/win_usermanual_en.html#NetworkPane.

**Reply by brunobaudry** [reply]
Thanks a lot Damian for your answer. Somehow "embedded content" did not ring the bell when reading the doc. Work fine indeed.
